pages/store/[nftClassId]/index.vue

```diff
@@ -178,16 +178,41 @@
                           class="text-sm font-semibold"
                           v-text="$t('product_page_sold_out_button_label')"
                         />
-                        <span v-else>
-                          <span
-                            class="text-xs mr-0.5"
-                            v-text="item.currency"
-                          />
-                          <span
-                            class="font-semibold"
-                            v-text="item.formattedPrice"
-                          />
-                        </span>
+                        <div
+                          v-else
+                          class="flex flex-col items-end text-right"
+                        >
+                          <template v-if="item?.discountedPrice">
+                            <div class="flex flex-nowrap items-center text-green-600 font-semibold ">
```

Comment:
Prefer using `<span/>` for inline layout element
```suggestion
                            <span class="flex flex-nowrap items-center text-green-600 font-semibold">
```